.github/workflows/dotnet-core.yml

```diff
@@ -1,10 +1,8 @@
 name: .NET
 
 on:
-  push:
-    branches: [ master ]
-    paths-ignore:
-    - '**.md'
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
please add a comment about "human readable schedule". like once a day or once a week.

Reply:
just added. thanks for reviewing.